components/item/AwesomeSwiper.vue

```diff
@@ -44,7 +46,7 @@
 <script>
   import { Swiper, SwiperSlide } from 'vue-awesome-swiper';
   import 'swiper/css/swiper.css';
```

Comment:
ok moved to css

Reply:
this actually causes an error on the unit tests, it works fine while running the app but on testing it can't locate the css file, tried a few solutions for this but the best solution I found was putting it back as module import within script

Reply:
Fair enough.